client/rest/src/test/java/org/elasticsearch/client/RestClientIntegTests.java

```diff
@@ -229,6 +229,18 @@ public void testPathPrefix() throws IOException {
         }
     }
 
+    public void testPath() throws IOException {
+        Response esResponse;
+        for (final String method : getHttpMethods()) {
+            try {
+                esResponse = restClient.performRequest(method, null);
```

Comment:
Thanks for your review.
Would it be better if we throw a NPE with some message like we do for pathPrefix?
In RestClientBuilder#setPathPrefix() we have:
```Objects.requireNonNull(pathPrefix, "pathPrefix must not be null");```